features/FEATURE_BLE/ble/generic/GenericGap.h

```diff
@@ -302,10 +335,12 @@ class GenericGap : public ::Gap,
 
     void on_unexpected_error(const pal::GapUnexpectedErrorEvent &e);
 
-    pal::own_address_type_t get_own_address_type();
+    pal::own_address_type_t get_own_address_type(bool central_not_peripheral, bool non_connectable_or_scan_request);
```

Comment:
maybe enums would be clearer? not sure what the intended meaning of a bool non_connectable_or_scan_request is

Reply:
nevermind, now I know what they mean, maybe just rename the first one to bool is_central, the second one is a mouthful but is exactly what it sounds like